public/components/SponsorshipEdit/SponsorEdit.react.js

```diff
@@ -37,6 +38,9 @@ export default class SponsorEdit extends React.Component {
       return false;
     }
 
+    const logoWidth = this.props.paidContentTagType === PAID_HOSTEDCONTENT_TYPE.value ? false : 140;
+    const logoHeight = this.props.paidContentTagType === PAID_HOSTEDCONTENT_TYPE.value ? false : 90;
```

Comment:
I guess this works, but it looks kind of ghastly when you're used to stronger typing.